server/serverless.yml

```diff
@@ -26,6 +26,12 @@ functions:
       - http:
           path: /home
           method: any
+  viewProfile:
+    handler: profile.getProfile
+    events:
+      - http:
+          path: /viewProfile
```

Comment:
Not sure if this is the endpoint exposed to users. If it is, we can make this more RESTful (same as earlier comment). If not, we should discuss during standup to understand the difference between this endpoint and those exposed to users through the app. Consistency may be desired for APIs.

Reply:
These config are only for our react app to call our lambda function. This is not visible to the user